Project/app/src/main/java/org/ionproject/android/SharedViewModel.kt

```diff
@@ -0,0 +1,21 @@
+package org.ionproject.android
+
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModel
+import org.ionproject.android.common.model.ClassSummary
+import org.ionproject.android.common.model.CourseSummary
+
```

Comment:
This class materializes a contract, right? It deserves to be documented. Please do not forget to document appropriately all contracts that are defined within the application (with KDoc).